src/windows/View.tsx

```diff
@@ -182,17 +205,30 @@ export class View extends ViewCommon implements React.ChildContextProvider<ViewC
         if (props.onKeyPress) {
 
             // Define the handler for "onKeyDown" on first use, it's the safest way when functions
-            // called from super constructors are involved.
-            if (!this._onKeyDown) {
-                this._onKeyDown =  (e: Types.SyntheticEvent) => {
-                    if (this.props.onKeyPress) {
-                        // A conversion to a KeyboardEvent looking event is needed
-                        this.props.onKeyPress(EventHelpers.toKeyboardEvent(e));
-                    }
-                };
+            // called from super constructors are involved. Ensuring nothing happens here if there 
+            // is a tabIndex else KeyDown is handled twice, in _onFocusableKeyDown as well. 
```

Comment:
Do you mean "if a tabIndex is specified" rather than "is a tabIndex"?

Reply:
yes, fixed.